### PR TITLE
updated Firebase SDKs

### PR DESCRIPTION
**Pull request by suntrix**

### CODE REVIEW

**Comment by suntrix** [comment]
@nbransby / @Reedyuk can you take a look?

File: firebase-config/src/jsTest/kotlin/dev/gitlive/firebase/remoteconfig/FirebaseRemoteConfig.kt [hunk 1]
```diff
@@ -9,8 +9,8 @@ actual fun runTest(test: suspend () -> Unit) = GlobalScope
     .promise {
         try {
             test()
-        } catch (e: dynamic) {
-            (e as? Throwable)?.log()
```
Comment (nbransby) [inline firebase-config/src/jsTest/kotlin/dev/gitlive/firebase/remoteconfig/FirebaseRemoteConfig.kt:13]: Throwable maps to Error js so this won't catch other non-errors that might be thrown (I observed the firebase libs throwing non-errors)

Comment (suntrix) [reply firebase-config/src/jsTest/kotlin/dev/gitlive/firebase/remoteconfig/FirebaseRemoteConfig.kt:13]: I'll revert that, thx!

Comment (suntrix) [reply firebase-config/src/jsTest/kotlin/dev/gitlive/firebase/remoteconfig/FirebaseRemoteConfig.kt]: @nbransby done

**Comment by nbransby** [comment]
as you can see from this PR https://github.com/GitLiveApp/firebase-kotlin-sdk/pull/224/files we actually removed that ktx dependency intentionally.

my preference would be to remove all the ktx dependencies - it pulls in extra dependencies aka extra bloat for the end user of this library which they don't gain from in anyway and we don't really need to use the ktx extensions internally in the lib

**Comment by suntrix** [comment]
@nbransby I removed all -ktx dependencies & rebased on top of master

**Comment by suntrix** [comment]
@nbransby / @Reedyuk can you take a look?

**Review by nbransby — APPROVED** [review]

**Pull request merged**